ci-operator/config/coreos/rpm-ostree/OWNERS

```diff
@@ -0,0 +1,14 @@
+approvers:
```

Comment:
These need `reviewers` so that we could LGTM PRs here

Reply:
Approvers get used when no reviewers get listed